rtb/glossario_ver3.2.0.typ

```diff
@@ -8,6 +8,12 @@
   titolo: "Glossario",
   verificatori: ("Andrea Perozzo", "Andrea Precoma", "Davide Picello"),
   versioni: (
+    "3.3.0",
+    "28/11/2024",
+    "Davide Picello",
+    "Riccardo Milan",
+    [- Aggiunto il termine "Docker"],
```

Comment:
aggiunto anche il termine Deep dive

Reply:
"Deep dive" si vede dalla tabella delle modifiche del glossario che è stato aggiunto in data 23/11/2024

Reply:
ok